### DISCUSSION TITLE
What do negative values for reaction rates imply in PorousFlow?

### DISCUSSION THREAD

**Discussion by ghost** [idaholab/moose#25697] [Q&A Modules: Porous Flow]
Hello, 
I recently simulated a simple dissolution problem using the PorousFlowAqueousPreDis Kernel. However, I was surprised to see that my dissolution rates were negative. According to the PorousFlow documentation, I expected positive values for dissolution and negative values for precipitation. Is this some type of error in my simulation? I tried convincing myself by running the PorousFlow dissolution_limited.i test file, and again, I observed negative values for that problem. Does it mean negative values correspond to dissolution? Could someone please clarify this? Thanks.

**Comment by cpgr** [comment]
So, is the mineral actually dissolving when the reaction rate is negative? And how are you checking the reaction rate?

**Reply by ghost** [reply]
Hi @cpgr, Yes. The mineral is dissolving while the reaction rate is negative.I’m getting the reaction rate from `PorousFlowPropertyAux` Kernel as follows:

  ```
[AuxVariables]
[diss_rate_per_sec]
    family = MONOMIAL
    order = CONSTANT
  []
[]

[AuxKernels]
  [diss_rate_per_sec]
    type = PorousFlowPropertyAux
    property = mineral_reaction_rate
    mineral_species = 0
    variable = diss_rate_per_sec
  []
[]  

```
Here is a graph of my reaction rate with the dissolving mineral:
![simple_dis](https://github.com/idaholab/moose/assets/146151960/fd550b91-314c-4ccd-9701-b8e6e873784b)
